### PR TITLE
MM-56540 - Add LiveCaptions message

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Add LiveCaptions necessary for live captions feature

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-56540

### CODE REVIEW

File: lib/types/types.d.ts [hunk 1]
```diff
@@ -89,7 +89,13 @@ export type UserRemovedData = {
     channel_id?: string;
     remover_id: string;
 } & BaseData;
-export type WebsocketEventData = EmptyData | HelloData | CallStartData | UserDisconnectedData | UserConnectedData | UserMutedUnmutedData | UserVoiceOnOffData | UserScreenOnOffData | UserRaiseUnraiseHandData | EmojiData | UserReactionData | CallHostChangedData | CallRecordingStateData | UserState | UserDismissedNotification | CallStateData | JobStopData | UserRemovedData;
+export type LiveCaptionData = {
+    channelID: string;
+    user_id: string;
+    session_id: string;
+    text: string;
```
Comment (streamer45) [inline lib/types/types.d.ts:96]: Will we ever need a language field or is it something we should describe at a higher level?

Comment (cpoile) [reply lib/types/types.d.ts:96]: I've been thinking about it, and afaik I don't think so. Unless we eventually have a translation function as well, which I can't see anytime soon.

File: lib/types/types.d.ts [hunk 2]
```diff
@@ -89,7 +89,13 @@ export type UserRemovedData = {
     channel_id?: string;
     remover_id: string;
 } & BaseData;
-export type WebsocketEventData = EmptyData | HelloData | CallStartData | UserDisconnectedData | UserConnectedData | UserMutedUnmutedData | UserVoiceOnOffData | UserScreenOnOffData | UserRaiseUnraiseHandData | EmojiData | UserReactionData | CallHostChangedData | CallRecordingStateData | UserState | UserDismissedNotification | CallStateData | JobStopData | UserRemovedData;
+export type LiveCaptionData = {
+    channelID: string;
```
Comment (streamer45) [inline lib/types/types.d.ts:93]: If it's not too painful I'd start using `channel_id` for new types to eventually reach case consistency.

**Review by streamer45 — APPROVED** [review]

File: lib/types/types.d.ts (hunk 2 above)
Comment (cpoile) [reply lib/types/types.d.ts]: Thanks, changed.

**Review by streamer45 — APPROVED** [review]

**Pull request merged**